src/app/admit-card/page.tsx

```diff
@@ -63,6 +72,22 @@ export default function Page() {
     }
   };
 
+  const sendOTP = async () => {
```

Comment:
What does this code do? We're not sending an OTP right now I think, it's just doing a sleep for 2.5 seconds? Why merge it into main?